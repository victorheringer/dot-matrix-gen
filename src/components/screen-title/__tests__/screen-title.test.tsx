import { render } from "@testing-library/react";
import { ScreenTitle } from "../index";

describe("ScreenTitle", () => {
  it("should render correctly", () => {
    const tree = render(<ScreenTitle>Test</ScreenTitle>).asFragment();
    expect(tree).toMatchSnapshot();
  });
});
