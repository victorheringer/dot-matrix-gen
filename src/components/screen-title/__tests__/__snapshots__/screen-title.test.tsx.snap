// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScreenTitle should render correctly 1`] = `
<DocumentFragment>
  <h1
    class="sc-bczRLJ eBVCLp"
  >
    Test
  </h1>
</DocumentFragment>
`;
